### ISSUE TITLE
API Basics: Still linking to MDN local library

### ISSUE THREAD

**Issue by damon314159** [TheOdinProject/curriculum#28586]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

I didn't see anyone report this already, but the API basics lesson refers back to the local library tutorial in the introduction

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/nodejs-api-basics

**Comment by MaoShizhong** [comment]
Nice catch, that was a sneaky one.

## Acceptance criteria

- [ ] The 4th intro paragraph in Node API Basics is reworded to refer to the Routes lesson's library routes example. The link also needs replacing with the Routes lesson link.

Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

**Comment by MevWebDev** [comment]
I want to be assigned to the issue

**Comment by MaoShizhong** [comment]
@MevWebDev assigned this to you. I see you've already opened a PR for this before assignment. Please note the following from my message above:

> Please comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

There's no guarantee that you'll be assigned, so to avoid the chance that you end up doing unnecessary work, [please don't open PRs to close an issue if you have not yet been assigned by a maintainer](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md#being-assigned-an-issue).
